llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -610,6 +599,15 @@ def IREELinalgExt_AttentionOp : IREELinalgExt_Op<"attention",
     ShapedType getOutputType() {
       return getOutput().getType().cast<ShapedType>();
     }
+    ShapedType getMaxType() {
+      return getMax().getType().cast<ShapedType>();
+    }
+    ShapedType getSumType() {
+      return getSum().getType().cast<ShapedType>();
+    }
+    ArrayRef<ShapedType> getResultTypes() {
```

Comment:
I dont think you need this method. The base `Operation` class has a `getResultTypes` method that should do the same.